contracts/actions/bprotocol/BprotocolLiquitySPWithdraw.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import "../../interfaces/bprotocol/IBAMM.sol";
+import "../ActionBase.sol";
+import "./helpers/BprotocolLiquitySPHelper.sol";
+import "../../utils/TokenUtils.sol";
+
+
+contract BprotocolLiquitySPWithdraw is ActionBase, BprotocolLiquitySPHelper {
+    using TokenUtils for address;
+
+    struct Params {
+        address to;
+        address lqtyTo;
+        uint256 amount;
```

Comment:
amount is non specific, given the user might think it's lusdAmount and it's lpAmount/shareAmount